src/ElectronNET.API/Runtime/StartupManager.cs

```diff
@@ -167,11 +167,11 @@ private BuildInfo GatherBuildInfo()
 
                 if (isSingleInstance?.Length > 0 && bool.TryParse(isSingleInstance, out var isSingleInstanceActive) && isSingleInstanceActive)
                 {
-                    buildInfo.ElectronSingleInstance = "yes";
+                    buildInfo.ElectronSingleInstance = "true";
```

Comment:
@FlorianRappl For what is the buildInfo used ? I could not find any meaningful usages of `buildInfo.ElectronSingleInstance`